### PR TITLE
Added a directory and a file.

### PR DESCRIPTION
**Pull request by jintaglee**
Updates to be merged into master

### CODE REVIEW

File: dir1/file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test line.
```
Comment (jintaglee) [inline dir1/file1.txt:1]: This looks O.K. but perhaps following may be better,
```suggestion
This is a test line added.
```

Comment (ghost) [inline dir1/file1.txt:1]: This look good.

**Review by jintaglee — COMMENTED** [review]
Just read my own PR again.

File: dir1/file1.txt (hunk 1 above)
Comment (jintaglee) [reply dir1/file1.txt:1]: Thanks. Can you merge the changes, please?

**Pull request merged**